src/Java.Interop.Dynamic/Tests/Java.Interop.Dynamic-Tests.csproj

```diff
@@ -36,9 +36,6 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\..\lib\NUnit-2.6.3\bin\nunit.framework.dll</HintPath>
```

Comment:
...what? `Java.Interop.Dynamic-Tests` doesn't reference `nunit.framework`?

What's odd is that (1) it does here, and (2) this PR built with the PR builder, so I am very confused:

```
$ monodis --assemblyref bin/TestDebug/Java.Interop.Dynamic-Tests.dll
...
3: Version=2.6.3.13283
	Name=nunit.framework
	Flags=0x00000000
	Public Key:
0x00000000: 96 D0 9A 1E B7 F4 4A 77 
```